### PR TITLE
Es6 projects commercial modules hosted

### PR DESCRIPTION
**Pull request by rich-nguyen**
Es6 conversions for my hosted files, plus two other small modules. 

## Tested in CODE?
yeah will do

### CODE REVIEW

**Review by lps88 — COMMENTED** [review]
Just testing around this, found one small issue on the gallery page. 

Go to the second image and then back to the top, and the CTA appears over the top:

image

There is a mechanism for showing the CTA on intermediate slides, which is probably what's broken, but I'm pretty sure we don't have any way of actually setting that up in composer, so feel free to remove the functionality altogether if that's easier!

**Review by lps88 — APPROVED** [review]
Looks good, are you planning on converting the unit tests too? Maybe in a separate PR?

**Comment by rich-nguyen** [comment]
Yeah, definitely aiming to move the tests over in another pr.

File: static/src/javascripts/projects/commercial/modules/high-merch.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+// @flow
+import config from 'lib/config';
+import fastdom from 'lib/fastdom-promise';
+import ab from 'common/modules/experiments/ab';
+import { addSlot } from 'commercial/modules/dfp/add-slot';
+import createSlot from 'commercial/modules/dfp/create-slot';
+import trackAdRender from 'commercial/modules/dfp/track-ad-render';
+import commercialFeatures from 'commercial/modules/commercial-features';
+
+const isLuckyBastard = () => {
```
Comment (sndrs) [inline static/src/javascripts/projects/commercial/modules/high-merch.js]: 🍀

**Pull request merged**